src/frontend/web_application/locale/fr/messages.json

```diff
@@ -279,13 +283,12 @@
   "message.from": "From:",
   "message.img.postal-card": "Ce message est comme une carte postale",
   "message.img.sealed-envelope": "Ce message est comme une enveloppe scellée",
+  "message.img.simple-envelope": "Ce messsage est comme une enveloppe simple",
   "message.participants.me": "Moi",
   "message.pi.comportment": "Expéditeur",
   "message.pi.context": "Départ",
-  "message.pi.description.metaphor.bad": "Dans la vraie vie, ce message serait plus ou moins l'équivalent d'une carte postale visible par tous.",
-  "message.pi.description.metaphor.good": "Dans la vraie vie, ce message serait plus ou moins l'équivalent d'une lettre sous enveloppe.",
-  "message.pi.description.metaphor.super": "Dans la vraie vie, ce message serait plus ou moins l'équivalent d'une lettre cachetée portée par un messager fiable.",
-  "message.pi.description.metaphor.ugly": "Dans la vraie vie, ce message serait plus ou moins l'équivalent d'une carte postale visible par tous.",
+  "message.pi.description.metaphor.encrypted": "Dans la vraie vie, ce message serait plus ou moins l'équivalent d'une lettre sous enveloppe.",
+  "message.pi.description.metaphor.unencrypted": "Dans la vraie vie, ce message serait plus ou moins l'équivalent d'une carte postale visible par tous.",
```

Comment:
Since Internet is now real life (unlike the 90') what do you think about the following formulation?

```diff
--- Dans la vraie vie
+++ Dans la vie courante
```